SparkPost/2web2ui
# SE-378: Add Metric Tag to Recent Incidents Cards on Alerts page

### What Changed
 - Changed "Recent Alerts" to "Recent Incidents"
 - Moved triggered time to bottom section of card
 - Removed the "View Details"/Eye Icon and made the alert name the link to the details
 - Added Metric tag to the bottom of the top section.

### How To Test
 - View /alerts page and ensure the following:
   - The Alert name is a link that goes to the details page
   - The Metric tag is displayed (in human readable form, not "monthly_spending_limit", etc.)
   - The TimeAgo is in the bottom section, aligned to the left
